app/actions.js

```diff
@@ -522,9 +524,9 @@ const initializeCabal = ({ addr, username, settings }) => async dispatch => {
     }
   ]
   cabalDetailsEvents.forEach((event) => {
-    cabalDetails.on(event.name, (data) => {
+    cabalDetails.on(event.name, throttle((data) => {
       event.action(data)
-    })
+    }), 2000)
```

Comment:
i think 2 seconds is a bit excessive for throttling if applied uniformly for all events. in particular i think `status-message` events will appear to be sluggish.

maybe lower the throttle amount to 200ms? each entry in `cabalDetailsEvents` could have an optional `throttleDelay` property which, if present, is used instead of the 200ms default?

wdyt @nikolaiwarner?

Reply:
ha! i knew you'd say that! good idea!